src/prototype_room_mineral.js

```diff
@@ -1,5 +1,17 @@
 'use strict';
 
+Room.prototype.isMineralInStorage = function() {
+  let mineralInStorage = false;
+  for (let resource in this.storage.store) {
+    if (resource === 'energy' || resource === 'power') {
+      continue;
+    }
+    mineralInStorage = true;
+    break;
+  }
+  return mineralInStorage;
```

Comment:
Object.keys(this.storage.store).some(r => r !== RESOURCE_ENERGY && r !== RESOURCE_POWER)